flink/v1.15/flink/src/test/java/com/netease/arctic/flink/util/MockEnvironmentBuilder.java

```diff
@@ -0,0 +1,198 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.flink.util;
+
+import org.apache.flink.api.common.ExecutionConfig;
+import org.apache.flink.api.common.JobID;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.runtime.externalresource.ExternalResourceInfoProvider;
+import org.apache.flink.runtime.io.disk.iomanager.IOManager;
+import org.apache.flink.runtime.io.disk.iomanager.IOManagerAsync;
+import org.apache.flink.runtime.jobgraph.JobVertexID;
+import org.apache.flink.runtime.memory.MemoryManager;
+import org.apache.flink.runtime.memory.MemoryManagerBuilder;
+import org.apache.flink.runtime.metrics.groups.TaskMetricGroup;
+import org.apache.flink.runtime.metrics.groups.UnregisteredMetricGroups;
+import org.apache.flink.runtime.operators.testutils.MockEnvironment;
+import org.apache.flink.runtime.operators.testutils.MockInputSplitProvider;
+import org.apache.flink.runtime.state.TaskStateManager;
+import org.apache.flink.runtime.state.TestTaskStateManager;
+import org.apache.flink.runtime.taskexecutor.GlobalAggregateManager;
+import org.apache.flink.runtime.taskmanager.TaskManagerRuntimeInfo;
+import org.apache.flink.runtime.throughput.ThroughputCalculator;
+import org.apache.flink.runtime.util.TestingTaskManagerRuntimeInfo;
+import org.apache.flink.runtime.util.TestingUserCodeClassLoader;
+import org.apache.flink.util.UserCodeClassLoader;
+
+public class MockEnvironmentBuilder {
+  private String taskName = "mock-task";
+  private MockInputSplitProvider inputSplitProvider = null;
+  private int bufferSize = 16;
+  private TaskStateManager taskStateManager = new TestTaskStateManager();
+  private GlobalAggregateManager aggregateManager = new TestGlobalAggregateManager();
+  private Configuration taskConfiguration = new Configuration();
+  private ExecutionConfig executionConfig = new ExecutionConfig();
+  private int maxParallelism = 1;
+  private int parallelism = 1;
+  private int subtaskIndex = 0;
+  private UserCodeClassLoader userCodeClassLoader = TestingUserCodeClassLoader.newBuilder().build();
+  private JobID jobID = new JobID();
+  private JobVertexID jobVertexID = new JobVertexID();
+  private TaskMetricGroup taskMetricGroup = UnregisteredMetricGroups.createUnregisteredTaskMetricGroup();
+  private TaskManagerRuntimeInfo taskManagerRuntimeInfo = new TestingTaskManagerRuntimeInfo();
+  private IOManager ioManager;
+  private MemoryManager memoryManager = this.buildMemoryManager(33554432L);
+  private ExternalResourceInfoProvider externalResourceInfoProvider;
+  private ThroughputCalculator throughputCalculator;
```

Comment:
If this calculator is useless in building MockEnvironmentArctic, why don't you remove it here?